torch/testing/_internal/codegen/random_topo_test.py

```diff
@@ -308,7 +308,7 @@ def runTest(seed, args):
                 print("val size: ", out.size())
     except Exception as err:
         raise Exception("Testing script failure with error message {0}\n\trepro by running:\n\t{1}".format(
-            str(err), reproString(seed, args)))
+            str(err), reproString(seed, args))) from None
```

Comment:
Please chain the exception here.
```suggestion
            str(err), reproString(seed, args))) from err
```